docs/shrc_diag_util.md

```diff
@@ -262,106 +282,170 @@ Line number information for classes in the cache is then shown with one of the f
 
 The following summary data is displayed:
 
-#### `baseAddress` and `endAddress`
- : The boundary addresses of the shared memory area that contains the classes.
-
-#### `allocation pointer`
- : The address where `ROMClass` data is currently being allocated in the cache.
-
-#### `cache layer`
-: The layer number that the cache stats relate to.
-
-#### `cache size` and `free bytes`
- : `cache size` shows the total size of the shared memory area in bytes, and `free bytes` shows the free bytes that remain.
+- `baseAddress` and `endAddress`
+    - The boundary addresses of the shared memory area that contains the classes.
+- `allocation pointer`
+    - The address where `ROMClass` data is currently being allocated in the cache.
+- `cache layer`
+    - The layer number that the cache stats relate to.
+- `cache size` and `free bytes`
+    - `cache size` shows the total size of the shared memory area in bytes
+    - `free bytes` shows the free bytes that remain.
```

Comment:
The free space is not necessarily all available for storing new classes. The cache contains separate areas for different data, and can reserve space for AOT and JIT data, as shown by subsequent summary data.